components/crypto_dot_com/browser/crypto_dot_com_json_parser_unittest.cc

```diff
@@ -0,0 +1,241 @@
+/* Copyright (c) 2020 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/crypto_dot_com/browser/crypto_dot_com_json_parser.h"
+
+#include "brave/components/crypto_dot_com/browser/crypto_dot_com_service.h"
```

Comment:
Include seems unused here

Reply:
type definitions are grabbed from this file